exonum-java-binding-common/src/test/java/com/exonum/binding/common/serialization/json/StoredConfigurationGsonSerializerTest.java

```diff
@@ -51,6 +82,35 @@ void nullPointerException() {
         () -> StoredConfigurationGsonSerializer.fromJson(jsonConfiguration));
   }
 
+  @Test
+  void readConfiguration() {
+    StoredConfiguration configuration = StoredConfigurationGsonSerializer
+        .fromJson(CONFIG_EXAMPLE);
+
+    assertThat(configuration, notNullValue());
+    assertThat(configuration.previousCfgHash(),
+        is(HashCode
+            .fromString("0000000000000000000000000000000000000000000000000000000000000000"))
+    );
+    assertThat(configuration.actualFrom(), is(0L));
+
+    assertThat(configuration.consensusConfiguration(), notNullValue());
+    assertThat(configuration.consensusConfiguration().maxMessageLen(), is(1048576));
```

Comment:
Please extract `configuration.consensusConfiguration()` in a variable: 
```
var consensusConfiguration = configuration.consensusConfiguration();
```
to simplify the code. 

Alternatively, this code can build an expected configuration and assert equality:
```
assertThat(configuration.consensusConfiguration(), equalTo(expectedConfiguration));
```

Reply:
Thanks, will extract variable.